src/anndata/io.py

```diff
@@ -27,6 +29,11 @@ def write_zarr(*args, **kw):
     return write_zarr(*args, **kw)
 
 
+if os.environ.get("READTHEDOCS", "") != "True":
```

Comment:
Is this boolean right?

Reply:
This is correct: https://docs.readthedocs.io/en/stable/reference/environment-variables.html#envvar-READTHEDOCS

Reply:
But now the build is failing because this READTHEDOCS not equal to True? https://dev.azure.com/scverse/anndata/_build/results?buildId=8179&view=logs&j=60c6783d-556f-53a9-3d94-3b9ccc51ad8d&t=3522b173-13e1-56b6-93d8-ed20bad6f9bb&l=61

Reply:
Ah I misunderstood your question. The `!=` is *not* correct. Spicy autocomplete fails again, this should be a `==`